goldrush_path/goldrush_path.cpp

```diff
@@ -152,10 +156,32 @@ fill_bit_vector(const std::string& input_file,
       }
       continue;
     }
+#pragma omp atomic
+    ++num_passed_reads;
     multiLensfrHashIterator itr(record.seq, spaced_seeds);
     miBFCS.insertBV(itr);
   }
 
+  if (verbose) {
+    std::cerr << "num_passed_reads: " << num_passed_reads << "\n"
+              << "num_reads: " << num_reads << "\n"
+              << "num_reads - num_passed_reads: "
+              << num_reads - num_passed_reads << "\n"
+              << "num_reads - num_passed_reads / num_reads: "
+              << "\n"
+              << (double)(num_reads - num_passed_reads) / num_reads
+              << std::endl;
+  }
+
+  if (num_passed_reads == 0) {
+    std::cerr
+      << "Error: no reads passed the Phred score and min length requirements"
+      << "\n"
+      << "Try again with a lower Phred score (-P) or lower min length (-m)"

```

Comment:
Should we have the parameters formatted like the makefile parameters vs. the goldrush-path executable parameters? Just wondering if the more common use case would be a user adjusting the makefile

Reply:
Sure, fine by me